StubConfigTest.testConfigureDeadlineNanoTime is flaky

```
io.grpc.stub.StubConfigTest > testConfigureDeadlineNanoTime FAILED
    java.lang.AssertionError: 20000000 < 45632697
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.assertTrue(Assert.java:41)
        at io.grpc.stub.StubConfigTest.testConfigureDeadlineNanoTime(StubConfigTest.java:116)
```

As seen in (may have been restarted):
https://travis-ci.org/grpc/grpc-java/jobs/121487417
https://travis-ci.org/grpc/grpc-java/jobs/121488444 (20000000 < 24579334)


https://travis-ci.org/grpc/grpc-java/jobs/118573602, which was before assertEquals float vs double fix, so it had:
```
io.grpc.stub.StubConfigTest > testConfigureDeadlineNanoTime FAILED
    java.lang.AssertionError: expected:<1.68848953E13> but was:<1.68849184E13>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:743)
        at org.junit.Assert.assertEquals(Assert.java:519)
        at org.junit.Assert.assertEquals(Assert.java:609)
        at io.grpc.stub.StubConfigTest.testConfigureDeadlineNanoTime(StubConfigTest.java:114)
```

--- a/interop-testing/src/test/java/io/grpc/stub/StubConfigTest.java
+++ b/interop-testing/src/test/java/io/grpc/stub/StubConfigTest.java
@@ -111,7 +111,7 @@ public class StubConfigTest {
     TestServiceGrpc.TestServiceBlockingStub reconfiguredStub = stub.withDeadlineNanoTime(deadline);
     // New altered config
     assertNotNull(reconfiguredStub.getCallOptions().getDeadlineNanoTime());
-    long maxDelta = MILLISECONDS.toNanos(20);
+    long maxDelta = MILLISECONDS.toNanos(30);
     long actualDelta = Math.abs(reconfiguredStub.getCallOptions().getDeadlineNanoTime() - deadline);
     assertTrue(maxDelta + " < " + actualDelta, maxDelta >= actualDelta);
     // Default config unchanged

